modules/service-fips.sh

```diff
@@ -50,6 +54,69 @@ fips_disable() {
     not_supported 'Disabling FIPS'
 }
 
+fips_updates_enable() {
+    local token="$1"
+    local bypass_prompt="$2"
+    local fips_configured=0
+    local fips_kernel_version=0
+    local result=0
+
+    check_token "$FIPS_UPDATES_REPO_URL" "$token"
+    _fips_updates_is_enabled || result=$?
+    if [ $result -eq 0 ]; then
+        error_msg "FIPS-UPDATES repository is already enabled."
+        return;
+    fi
```

Comment:
Instead of just returning, shouldn't we perhaps `error_exit service_already_enabled`? Like we do for the non-updates fips case a few lines up in the same module:
```
fips_enable() {
    local token="$1"

    _fips_check_installed || error_exit service_already_enabled

    check_token "$FIPS_REPO_URL" "$token"
...
````

Reply:
fips-update is not a separate service. FIPS_SERVICE_TITLE is defined only for FIPS service. On a fips enabled machine trying to enable-fips again shows "Canonical FIPS 140-2 Modules is already enabled". Since FIPS-UPDATES is not a service and title is not defined, it does not show any error message on exit. So I added a separate error_msg and return.

Reply:
I think you should be able do do what @panlinux suggests, IOW `_fips_updates_is_enabled || error_exit service_already_enabled`. Even if `fips-updates` is not a separate service, the error still makes sense to me in this context.
If you want to show the error you can:

```
if _fips_updates_is_enabled; then
    error_msg "FIPS-UPDATES repository is already enabled."
    error_exit service_already_enabled
fi
```